orderer/common/msgprocessor/expiration_test.go

```diff
@@ -22,12 +22,14 @@ import (
 
 //go:generate counterfeiter -o mocks/config_resources.go --fake-name Resources . configResources
 
+// TODO delete?
 type configResources interface {
 	channelconfig.Resources
 }
 
 //go:generate counterfeiter -o mocks/orderer_config.go --fake-name OrdererConfig . ordererConfig
 
+// TODO delete?
```

Comment:
no, it is an auxiliary interface used to create the mock

Reply:
Check the mocks that remain, you'll find that `mocks/sig_filter_support.go` is never used. delete it and also track down the generate statement that created it and remove it as well

Reply:
Done!